here-naksha-lib-psql/src/main/java/com/here/naksha/lib/psql/PostgresSession.java

```diff
@@ -630,6 +647,8 @@ <FEATURE, CODEC extends FeatureCodec<FEATURE, CODEC>> Result executeWrite(
       final PreparedStatement stmt = prepareStatement(
           "SELECT r_op, r_id, r_uuid, r_type, r_ptype, r_feature, ST_AsEWKB(r_geometry), r_err\n"
               + "FROM nk_write_features(?,?,?,?,?,?,?,?,?);");
+      final int SIZE = writeRequest.features.size();
+      String collection_id = null;
```

Comment:
Is it possible to retain collection_id as final and assign the value here itself?

Reply:
Fixed It.